infra-as-code/bicep/modules/vwanConnectivity/parameters/vwanConnectivity.parameters.all.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "parVirtualHubEnabled": {
+        "value": true
+      },
+      "parVPNGatewayEnabled":{
+          "value": true
+      },
+      "parERGatewayEnabled":{
+        "value": true
+      },      
+      "parAzureFirewallEnabled": {
+          "value": true
+      },
+      "parNetworkDNSEnableProxy": {
+        "value": true
+    },      
+      "parCompanyPrefix":{
+          "value": "alz"
+      },
+      "parPublicIPSku":{
```

Comment:
```suggestion
      "parPublicIpSku":{
```

Reply:
Please see latest update to PR and resolve this conversation.